pkgs/development/python-modules/backports-zoneinfo/default.nix

```diff
@@ -27,8 +40,14 @@ buildPythonPackage rec {
     pytestCheckHook
   ];
 
-  # unfortunately /etc/zoneinfo doesn't exist in sandbox, and many tests fail
-  doCheck = false;
+  disabledTests = [
+    # AssertionError: 'AEDT' != 'AEST'
+    "test_folds_and_gaps"
+    # backports.zoneinfo._common.ZoneInfoNotFoundError: 'No time zone found with key Eurasia/Badzone'
+    "test_bad_keys"
+    # AssertionError: 0 != 1 : (datetime.datetime(1917, 3, 25, 2, 0, 1, tzinfo=backports.zoneinfo.ZoneInfo(key='Australia/Sydney')), datetime.datetime(1917, 3, 24, 15, 0, tzinfo=datetime.timezone.utc))
+    "test_folds_from_utc"
+  ];
```

Comment:
I honestly assume these tests fail since tzdata was updated since they were written.

AEDT is Australian Eastern Daylight Time
AEST is Australian Eastern Standard Time

and now we have an AssertionError in a comparsion with Australia/Sidney.